django/db/models/query.py

```diff
@@ -691,22 +721,35 @@ def values(self, *fields, **expressions):
         clone._iterable_class = ValuesIterable
         return clone
 
-    def values_list(self, *fields, flat=False):
+    def values_list(self, *fields, flat=False, named=False):
+        if flat and named:
+            raise TypeError("'flat' and 'named' can't be passed together.")
```

Comment:
For similarity with other messages (e.g. 'The nowait option cannot be used with skip_locked.') you might change 'passed' to 'used'.